csharp/GildedRose.cs

```diff
@@ -4,85 +4,102 @@ namespace csharp
 {
     public class GildedRose
     {
-        IList<Item> Items;
-        public GildedRose(IList<Item> Items)
+        IList<Item> items;
+        public GildedRose(IList<Item> items)
         {
-            this.Items = Items;
+            this.items = items;
         }
 
+        public int QualityInBounds(Item item)
```

Comment:
Probably a better name would be "BoundedQuality", otherwise I think it's implying that it's checking whether the quality is in bounds. Or maybe I did too many cinema interviews :))